setup.py

```diff
@@ -1,8 +1,13 @@
 from setuptools import setup, find_packages
 
 setup(
-    name = "regulations",
-    version = "0.1.0", 
-    license = "public domain", 
-    packages = find_packages()
+    name="regulations",
```

Comment:
The app has had the name `regulations` for a long time. I'm inclined to keep it that way for compatibility